<html>
<head>
  <title>I-TASSER results</title>
</head>

<script type="text/javascript" src="../../bin/jmol/JSmol.min.js"></script>
<script type="text/javascript" src="../../bin/jmol/Jmol2.js"></script>
<body bgcolor="#FFFFFF">
<font face="Times New Roman">
[<a href="http://zhanglab.ccmb.med.umich.edu">Home</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER">Server</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/queue.php">Queue</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/about.html">About</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/remove.html">Remove</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/statistics.html">Statistics</a>]
<br/>
<table border="0" style="table-layout:fixed;" width="100%" cellpadding="2" cellspaceing="0">
<tr><td>
  <h1 align="center">I-TASSER results for job id S760663</h1>
<center> (Click on <a href="S760663_results.tar.bz2" download>S760663_results.tar.bz2</a> to download the tarball file including all modeling results listed on this page. Click on <a href=http://zhanglab.ccmb.med.umich.edu/I-TASSER/annotation target=_blank>Annotation of I-TASSER Output</a> to read the instructions for how to interpret the results on this page. Model results are kept on the server for 60 days, there is no way to retrieve the modeling data older than 2 months)
</center><br>
</td></tr>
<tr><td>
   <div style="background:#6599FF;width:350px;">
   <table><tr><td style= "valign:middle;">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Submitted Sequence in <a href=http://zhanglab.ccmb.med.umich.edu/FASTA/ target="_blank">FASTA format</a></font>
   </td></tr></table>
   </div>
   <br/>
   <div style="position:relative;left:30px;">
   <table style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
   <tr><td colspan="2">>protein<br>
GAMGMKNAPLTLNFGSVRLPVSADGLLHAPTAQQQLGLTQSWEAALVEHGLPETYRDFGA<br/>
GPEAAVSVPDFVALAFALDTPEARRWQKRARELLARAMQGDVRVAAQIAERNPEPDARRW<br/>
LAARLESTGARRELLATVARHGGEGRVYGQLGSISNRTVLGKDSASVRQERGVKATRDGL<br/>
TSAELLRLAYIDTVTARAIQESEARGNAAILTLHEQVARSERQSWERAGQVQRVG<br/>
</td></tr>
   </table>
   </div>
</td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:300px;">
   <table><tr><td valign="middle">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted Secondary Structure</font>
   </td></tr></table>
   </div>
   <br/>
   <div style="position:relative;left:30px;">
    <table  border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td valign="bottom"><font="Arial"><b>Sequence</b></font></td><td BGCOLOR="#DEDEDE">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;140&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;160&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;180&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;200&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;220<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br/>GAMGMKNAPLTLNFGSVRLPVSADGLLHAPTAQQQLGLTQSWEAALVEHGLPETYRDFGAGPEAAVSVPDFVALAFALDTPEARRWQKRARELLARAMQGDVRVAAQIAERNPEPDARRWLAARLESTGARRELLATVARHGGEGRVYGQLGSISNRTVLGKDSASVRQERGVKATRDGLTSAELLRLAYIDTVTARAIQESEARGNAAILTLHEQVARSERQSWERAGQVQRVG</td></tr>
    <tr><td><font="Arial"><span title="Secondary Structure Prediction"><b>Prediction</b></span></font></td><td BGCOLOR="#D7D7D7">CCCCCCCCCCCCCCCC<font color="blue">SS</font>CCCCCC<font color="red">HHH</font>CC<font color="red">HHHHHHH</font>CCC<font color="red">HHHHHHHHH</font>CCCCCC<font color="red">HHH</font>CCCCCCCCCC<font color="red">HHHHHHHHHH</font>CCC<font color="red">HHHHHHHHHHHHHHHHHH</font>CCC<font color="red">HHHHHHHHHH</font>CCCCC<font color="red">HHHHHHHHHHHHHHHHHHHHHHHH</font>CCCCCC<font color="red">HHHHHHHHHHHHH</font>CCCC<font color="red">HHHHHHHH</font>CC<font color="red">HHH</font>CCCCC<font color="red">HHHHHHHHHHHHHHHHHHHH</font>CCCCCC<font color="red">HHHHHHHHHHHHHHHHHHHH</font>CCCCCCCC</td>
    <tr><td><font="Arial"><span title="Confidence Score of Prediction"><b>Conf.Score</b></span></font></td><td BGCOLOR="#ECECEC">9643135853000164545658530020448998746521058888733799974552888754378999999998469777799999999999987684403999998746785018999999999999999999999818876022334336667760777899998606102126898799999999999988776641665558999987899999999888623787889</td></tr>
    <tr><td><font="Arial"></td><td BGCOLOR="#ECECEC"><font color=red>H</font>:Helix; <font color=blue>S</font>:Strand; C:Coil</td><tr>
    </table>
  </div>
  </td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:300px;">
   <table><tr><td valign="middle">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted Solvent Accessibility</font>
   </td></tr></table>
   </div>
   <br/>
   <div style="position:relative;left:30px;">
    <table  border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td valign="bottom"><font="Arial"><b>Sequence</b></font></td><td BGCOLOR="#DEDEDE">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;140&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;160&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;180&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;200&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;220<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br/>GAMGMKNAPLTLNFGSVRLPVSADGLLHAPTAQQQLGLTQSWEAALVEHGLPETYRDFGAGPEAAVSVPDFVALAFALDTPEARRWQKRARELLARAMQGDVRVAAQIAERNPEPDARRWLAARLESTGARRELLATVARHGGEGRVYGQLGSISNRTVLGKDSASVRQERGVKATRDGLTSAELLRLAYIDTVTARAIQESEARGNAAILTLHEQVARSERQSWERAGQVQRVG</td></tr>
    <tr><td><font="Arial"><span title="Solvent Accessibility Prediction"><b>Prediction</b></span></font></td><td BGCOLOR="#D7D7D7">7434244032214233231223351303033124414146304411663614653552242352313263012002204443244024304520440353624044203734444444410342064352344025304734241431030232144313444354145344145536613322002311143134531565536334511543451244444426444625658</td><tr>
    <tr><td><font="Arial"></td><td BGCOLOR="#ECECEC">Values range from <b>0</b> (buried residue) to <b>9</b> (highly exposed residue)</td><tr>
    </table>
  </div>
  </td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:300px;">
   <table><tr><td valign="middle">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp; Predicted normalized B-factor</font>
   </td></tr></table>
   </div>
   <br/>

<font size=2>
(B-factor is a value to indicate the extent of the inherent thermal 

mobility of residues/atoms in proteins. In I-TASSER, this value is 

deduced from threading template proteins from the PDB in combination with the 

sequence profiles derived from sequence databases. The reported B-factor 

profile in the figure below corresponds to the normalized B-factor of the 

target protein, defined by B=(B'-u)/s, where B' is the raw B-factor 

value, u and s are respectively the mean and standard deviation of the raw 

B-factors along the sequence. 

<a href="lscore.txt" target="_blank"  >Click here to read more about predicted 

normalized B-factor</a>) 

</font> 

<br/> 

   <br/> 

   <div style="position:relative;left:30px;">
    <table  border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr> <td align =center > <img src=BFP.png>  </td>  </tr>
    </table>
  </div>
  </td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:450px;">
   <table><tr><td style= "valign:middle;">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Top 10 threading templates used by I-TASSER</font>
   </td></tr></table>
   </div>
   <br/>
   
<font size=2> 

(I-TASSER modeling starts from the structure templates identified by LOMETS from the PDB library. 

LOMETS is a meta-server threading approach containing multiple threading programs, 

where each threading program can generate tens of thousands of 

template alignments. I-TASSER only uses the templates of the highest significance in the threading alignments, 

the significance of which are measured by the Z-score, i.e. the difference between the raw and average scores 

in the unit of standard deviation. The templates in this section are the 10 best templates selected from 

the LOMETS threading programs. Usually, one template of the highest Z-score is selected from each threading program, where the threading programs are sorted by the average performance in the large-scale benchmark test experiments.) 

</font> 

   <br/> 

   <br/> 

   <div style="position:relative;left:30px;">
   <table style="font-size:13px;background:#F2F2F2;">
      <tr> <th align="left" valign="top"><font face="Arial">Rank</font></th> <th align="center" valign="top"><font face="Arial">PDB<br/>Hit</font></th><th align="center" valign="top"><font face="Arial">Iden1</font></th><th align="center" valign="top"><font face="Arial">Iden2</font></th><th valign="top"><font face="Arial">Cov</font></th><th valign="top"><font face="Arial">Norm.<br/>Z-score</font></th><th valign="top"><font face="Arial">Download<br/>Align.</font></th><td>&nbsp;</td><td BGCOLOR="#DEDEDE" style="font-family:Monospace;font-size:13px;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;140&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;160&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;180&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;200&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;220<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td></tr>
      <tr><td colspan="7"></td><th align="right" valign ="bottom" ><font face="Arial">Sec.Str<br/>Seq</font></th><td BGCOLOR="#D7D7D7" style="font-family:Monospace;font-size:13px;">CCCCCCCCCCCCCCCC<font color="blue">SS</font>CCCCCC<font color="red">HHH</font>CC<font color="red">HHHHHHH</font>CCC<font color="red">HHHHHHHHH</font>CCCCCC<font color="red">HHH</font>CCCCCCCCCC<font color="red">HHHHHHHHHH</font>CCC<font color="red">HHHHHHHHHHHHHHHHHH</font>CCC<font color="red">HHHHHHHHHH</font>CCCCC<font color="red">HHHHHHHHHHHHHHHHHHHHHHHH</font>CCCCCC<font color="red">HHHHHHHHHHHHH</font>CCCC<font color="red">HHHHHHHH</font>CC<font color="red">HHH</font>CCCCC<font color="red">HHHHHHHHHHHHHHHHHHHH</font>CCCCCC<font color="red">HHHHHHHHHHHHHHHHHHHH</font>CCCCCCCC<br/>GAMGMKNAPLTLNFGSVRLPVSADGLLHAPTAQQQLGLTQSWEAALVEHGLPETYRDFGAGPEAAVSVPDFVALAFALDTPEARRWQKRARELLARAMQGDVRVAAQIAERNPEPDARRWLAARLESTGARRELLATVARHGGEGRVYGQLGSISNRTVLGKDSASVRQERGVKATRDGLTSAELLRLAYIDTVTARAIQESEARGNAAILTLHEQVARSERQSWERAGQVQRVG</td></tr>       <tr><td align="center"><font face="Arial">1</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=7xge" target="_blank">7xgeA</a></td><td align ="center"><font face="Arial"> 0.12</font></td><td><font face="Arial"> 0.18</font></td><td><font face="Arial"> 0.61</font></td><td align="center"><font face="Arial"> 1.07</font></td><td align="center"><a href="threading1.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-------------------------------------------------------------------------------KKV<font color="#333333">A</font>KKAAIQ<font color="#333333">A</font>ARRITELA<font color="#00DCDC">Q</font>VL<font color="#0F820F">V</font>ELLKEALKLDLTQEM<font color="#145AFF">R</font>KK<font color="#0F820F">L</font>IE<font color="#145AFF">R</font>YAAAII<font color="#145AFF">R</font>GDINNAIYQAKVDSDQLDA<font color="#0F820F">L</font>LRALDEK<font color="#0F820F">V</font>AS<font color="#145AFF">K</font>AANQLGRLKRLDKDNG<font color="#666666">G</font>EEEKDRTAKWFEFEAR<font color="#333333">A</font>IE<font color="#0F820F">I</font>ALR-----L<font color="#333333">A</font>AIGDVFDLEKEW<font color="#145AFF">R</font>K-LKSR<font color="#666666">G</font>------</td></tr>
       <tr><td align="center"><font face="Arial">2</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=6w2v" target="_blank">6w2vA</a></td><td align ="center"><font face="Arial"> 0.11</font></td><td><font face="Arial"> 0.21</font></td><td><font face="Arial"> 0.88</font></td><td align="center"><font face="Arial"> 1.08</font></td><td align="center"><a href="threading2.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-------------------TDKEEVIEIVKELAELAKQSTDPNLVAEVVRALTEVAKTSTDT<font color="#E60A0A">E</font>LIREIIKVLLELASKL---------<font color="#145AFF">R</font>DPQAVLE<font color="#333333">A</font>L<font color="#00DCDC">Q</font>AVAEL<font color="#333333">A</font>REL<font color="#333333">A</font><font color="#E60A0A">E</font>KTGD<font color="#D69682">P</font>I<font color="#333333">A</font>KECAE<font color="#333333">A</font>VSAAAE<font color="#333333">A</font>VKKAADLLK<font color="#145AFF">R</font><font color="#8282D2">H</font>P<font color="#666666">G</font>SEAAQAA<font color="#0F820F">L</font>ELAKAAAEAVLIACLLALDYPKSDIAKKCIK<font color="#333333">A</font>ASEAAEEASKAAEE<font color="#333333">A</font>QRHPDSQKARDEIKEAS<font color="#00DCDC">Q</font>K<font color="#333333">A</font>EEVKERC<font color="#E60A0A">E</font><font color="#145AFF">R</font><font color="#333333">A</font>QEHPNA<font color="#666666">G</font></td></tr>
       <tr><td align="center"><font face="Arial">3</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=6qpi" target="_blank">6qpiA</a></td><td align ="center"><font face="Arial"> 0.08</font></td><td><font face="Arial"> 0.14</font></td><td><font face="Arial"> 0.93</font></td><td align="center"><font face="Arial"> 1.06</font></td><td align="center"><a href="threading3.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">--L<font color="#666666">G</font>YYTQM<font color="#0F820F">L</font>L<font color="#0F820F">L</font>AV<font color="#666666">G</font>VACFLYFGTLIFAVFMGVWVTLFLEF<font color="#B45AB4">W</font>KRRQA<font color="#E60A0A">E</font>LE<font color="#0F820F">L</font>CASAVF--------FWILLII<font color="#333333">A</font>SVIGITASIISFIIIMILNTIYEKVAFLFQFVNYYSSCFYIAFFKGE<font color="#0F820F">L</font>TTQ<font color="#0F820F">L</font>TIIMGGKAIWNNIQEVLLFYEYLEMIIQFGFV<font color="#FA9600">T</font>LFVASAPLLALVNNILEI<font color="#145AFF">R</font>VDAWQPIMQGI<font color="#333333">A</font>ILAV<font color="#0F820F">V</font><font color="#FA9600">T</font>NAM<font color="#0F820F">I</font>IAFTSDMWHV<font color="#0F820F">I</font>AAKLAFIIVM<font color="#E60A0A">E</font>HIIYSVK-------</td></tr>
       <tr><td align="center"><font face="Arial">4</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=6xns" target="_blank">6xnsA</a></td><td align ="center"><font face="Arial"> 0.10</font></td><td><font face="Arial"> 0.17</font></td><td><font face="Arial"> 0.59</font></td><td align="center"><font face="Arial"> 1.04</font></td><td align="center"><a href="threading4.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-------------------------------------------------------------------------------AKKLKTFLENL<font color="#145AFF">R</font>RH<font color="#0F820F">L</font>D<font color="#145AFF">R</font>LDKHIKQLRDILS<font color="#E60A0A">E</font>NPEDERVKDVIDLSER<font color="#FA9600">S</font>VRIVKTVIKIFEDS--VRKLLK<font color="#00DCDC">Q</font>INKEAEELAKSP<font color="#E60A0A">D</font>PEDLKRAVELAERA<font color="#E60A0A">D</font>PGSNLSKKA<font color="#0F820F">L</font>EI<font color="#0F820F">I</font>LRAA<font color="#333333">A</font>ELAKLPDPDAL<font color="#333333">A</font><font color="#333333">A</font>AARAASK<font color="#0F820F">V</font>QQE---------------</td></tr>
       <tr><td align="center"><font face="Arial">5</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=7jh6" target="_blank">7jh6A</a></td><td align ="center"><font face="Arial"> 0.12</font></td><td><font face="Arial"> 0.19</font></td><td><font face="Arial"> 0.63</font></td><td align="center"><font face="Arial"> 1.02</font></td><td align="center"><a href="threading5.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">------------------------------------------------------------------YLRELLK<font color="#0F820F">L</font>ELQ-AIKQYEKLRQTGD<font color="#E60A0A">E</font><font color="#0F820F">L</font>VQAFQR-----LRE<font color="#0F820F">I</font>FDKGDDDSLEQV<font color="#0F820F">L</font>EEIE<font color="#E60A0A">E</font>LIQKH<font color="#145AFF">R</font>Q<font color="#0F820F">L</font>ASELPKL------EL<font color="#00DCDC">Q</font>AIKQYREALEYVKLPVLAKI--LEDEEKHIEWLKEAAKQGDQW<font color="#0F820F">V</font>QRE<font color="#333333">A</font><font color="#0F820F">I</font>DKGDKDSLEQL<font color="#0F820F">L</font>EEL<font color="#E60A0A">E</font><font color="#00DCDC">Q</font>ALQKI<font color="#145AFF">R</font>ELT<font color="#E60A0A">E</font>KT<font color="#666666">G</font>------</td></tr>
       <tr><td align="center"><font face="Arial">6</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=7zo8" target="_blank">7zo8A</a></td><td align ="center"><font face="Arial"> 0.10</font></td><td><font face="Arial"> 0.20</font></td><td><font face="Arial"> 0.86</font></td><td align="center"><font face="Arial"> 1.01</font></td><td align="center"><a href="threading6.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">----LITM<font color="#D69682">P</font><font color="#0F820F">L</font>AWGTSNAL-----HT<font color="#0F820F">L</font>IR<font color="#333333">A</font>TDSLFT<font color="#0F820F">L</font>W<font color="#0F820F">L</font>EFMRQHLTT-----------------V<font color="#0F820F">V</font>ALATLIPV<font color="#333333">A</font>MTM<font color="#E60A0A">D</font>MRMSQLVMRKTKDGQ<font color="#333333">A</font>AVEKHHHKLFEHVSDTISNVSVVQSYNRIASE<font color="#FA9600">T</font>Q<font color="#333333">A</font>L<font color="#145AFF">R</font>DYAKNLENAQFPVLNWWA<font color="#0F820F">L</font>A<font color="#FA9600">S</font>GL<font color="#00DCDC">N</font><font color="#145AFF">R</font>MASTFSMVV<font color="#0F820F">V</font>LVLGAYFV<font color="#FA9600">T</font>KGQMRVGDVIAFIGFAQLMIGRLDQ-----IS<font color="#333333">A</font>FINQTVTARAKLEEFFQMEDATA---</td></tr>
       <tr><td align="center"><font face="Arial">7</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2tma" target="_blank">2tmaA</a></td><td align ="center"><font face="Arial"> 0.07</font></td><td><font face="Arial"> 0.16</font></td><td><font face="Arial"> 0.84</font></td><td align="center"><font face="Arial"> 1.01</font></td><td align="center"><a href="threading7.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">------------------------------D<font color="#333333">A</font>IKKKMKLDKEN<font color="#333333">A</font>LDRAEQAEADKKAAEDRSKQLEDELVSLQKKLKGTED<font color="#E60A0A">E</font>LDKYSEALKDAQEKLELAEKKATDAE<font color="#333333">A</font>DVASLNRRIQLVEEE<font color="#0F820F">L</font>DRAQE<font color="#145AFF">R</font>ATA<font color="#0F820F">L</font>QKLEEAEKAADESERGMKVIESRAQKDEEKMEI<font color="#00DCDC">Q</font><font color="#E60A0A">E</font>IQL<font color="#145AFF">K</font>EAKHIAED<font color="#333333">A</font>DRKYEEVARKLVIIESDLER<font color="#333333">A</font>EER<font color="#333333">A</font>ELSEGKCAELEE<font color="#E60A0A">E</font>IKTVTNN-------</td></tr>
       <tr><td align="center"><font face="Arial">8</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=8f7n" target="_blank">8f7nA</a></td><td align ="center"><font face="Arial"> 0.11</font></td><td><font face="Arial"> 0.17</font></td><td><font face="Arial"> 0.92</font></td><td align="center"><font face="Arial"> 0.65</font></td><td align="center"><a href="threading8.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-----------S<font color="#00DCDC">N</font>SVLKT<font color="#0F820F">L</font>SGFK<font color="#E60A0A">D</font>VYAQMNNFL<font color="#00DCDC">Q</font><font color="#00DCDC">Q</font>TTDRMLKD<font color="#333333">A</font>IVTQKETAAQVAGGEDELA<font color="#333333">A</font><font color="#333333">A</font>IRIDGLWT<font color="#0F820F">L</font>HEGEQRA<font color="#E60A0A">E</font>T<font color="#145AFF">R</font>ADLE<font color="#145AFF">R</font>LAAEQ<font color="#333333">A</font>KINEEAN<font color="#145AFF">R</font>LQYAVRKDELIAGTESV<font color="#0F820F">L</font>T<font color="#333333">A</font>TRRLSTSLTDIQIAA<font color="#333333">A</font>AFL<font color="#666666">G</font>TTSEESRKKLLDRFLAVQSNLTTL<font color="#145AFF">R</font>GIASGMSFF<font color="#E60A0A">D</font>QAALLPIIDGMKK<font color="#E60A0A">D</font>GLALVE<font color="#0F820F">I</font>TDKRTVEFE<font color="#333333">A</font>AGAAINEIWSDLTGFA<font color="#E60A0A">E</font>QQ-------</td></tr>
       <tr><td align="center"><font face="Arial">9</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1pv9" target="_blank">1pv9A</a></td><td align ="center"><font face="Arial"> 0.11</font></td><td><font face="Arial"> 0.24</font></td><td><font face="Arial"> 0.91</font></td><td align="center"><font face="Arial"> 0.67</font></td><td align="center"><a href="threading9.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;"><font color="#666666">G</font>TSPLGGGYIIVDGDEAT<font color="#0F820F">L</font><font color="#D69682">P</font>ELEYEMAKEESKLPVVKFKKFD<font color="#E60A0A">E</font>IYEILKNTELS<font color="#3232AA">Y</font>SMVEKSVKEFKKID<font color="#E60A0A">D</font>VIKDLRIIK<font color="#FA9600">T</font>K<font color="#E60A0A">E</font>EIEIIEK<font color="#333333">A</font>C<font color="#E60A0A">E</font>IADK<font color="#333333">A</font>VM---AAIEE<font color="#0F820F">I</font>----T<font color="#E60A0A">E</font>GKR---------EREVAAKVEYLMKMN<font color="#666666">G</font>A<font color="#E60A0A">E</font>KPAFDTISGHRSALPH<font color="#666666">G</font>VSDKRIERYNHSDI<font color="#FA9600">T</font><font color="#145AFF">R</font>GSPNEKQREIYEIVLEAQK<font color="#145AFF">R</font><font color="#333333">A</font>VEAAKP-<font color="#666666">G</font>MT-----AKELDSIA<font color="#145AFF">R</font>EIIKEY<font color="#666666">G</font>YGDYFI</td></tr>
       <tr><td align="center"><font face="Arial">10</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2jgd" target="_blank">2jgdA</a></td><td align ="center"><font face="Arial"> 0.22</font></td><td><font face="Arial"> 0.11</font></td><td><font face="Arial"> 0.14</font></td><td align="center"><font face="Arial"> 0.37</font></td><td align="center"><a href="threading10.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-------------------------------------------------------------------------------------------<font color="#E60A0A">E</font>A<font color="#0F820F">L</font>KQTYC<font color="#666666">G</font>PIGA--EYMH-ITSTEEK<font color="#145AFF">R</font><font color="#B45AB4">W</font>IQQ<font color="#145AFF">R</font>I<font color="#E60A0A">E</font>-------------------------------------------------------------------------------------------------------------</td></tr>
      <tr><td colspan="9" align="left">
        <table cellspacing="2" border="0" width="1000px" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top">(a)</td><td align="justify">All the residues are colored in black; however, those residues in template which are identical to the residue in the query sequence are highlighted in color. Coloring scheme is based on the property of amino acids, where polar are brightly coloured while non-polar residues are colored in dark shade. (<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/color.html" target="_blank">more about the colors used</a>)</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">Rank of templates represents the top ten threading templates used by I-TASSER.</td></tr>
        <tr><td valign="top">(c)</td><td align="justify">Ident1 is the percentage sequence identity of the templates in the threading aligned region with the query sequence.</td></tr>
        <tr><td valign="top">(d)</td><td align="justify">Ident2 is the percentage sequence identity of the whole template chains with  query sequence.</td></tr>
        <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of the threading alignment and is equal to the number of aligned residues divided by the length of query protein.</td></tr>
        <tr><td valign="top">(f)</td><td align="justify">Norm. Z-score is the normalized Z-score of the threading alignments. Alignment with a Normalized Z-score >1 mean a good alignment and vice versa.</td></tr>
        <tr><td valign="top">(g)</td><td align="justify">Download Align. provides the 3D structure of the aligned regions of the threading templates.</td></tr>
        <tr><td valign="top">(h)</td><td align="justify">The top 10 alignments reported above (in order of their ranking) are from the following threading programs:</td></tr>
        <tr><td>&nbsp;</td><td align ="left">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
1: FFAS-3D&nbsp;&nbsp;&nbsp;2: Neff-PPAS&nbsp;&nbsp;&nbsp;3: FFAS-3D&nbsp;&nbsp;&nbsp;4: FFAS-3D&nbsp;&nbsp;&nbsp;5: FFAS-3D&nbsp;&nbsp;&nbsp;6: FFAS-3D&nbsp;&nbsp;&nbsp;7: FFAS-3D&nbsp;&nbsp;&nbsp;8: SPARKS-X&nbsp;&nbsp;&nbsp;9: SP3&nbsp;&nbsp;&nbsp;10: HHSEARCH&nbsp;&nbsp;&nbsp;        </td></tr>
      </table>
      </td></tr></table>
   </div>
</td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:380px;">
   <table><tr><td style= "valign:middle;">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp; Top 5 final models predicted by I-TASSER </font>
   </td></tr></table>
   </div>
   <br/>
  
<font size=2>

(For each target, I-TASSER simulations generate a large ensemble of
structural conformations, called decoys. To select the final models, I-TASSER uses the SPICKER program to cluster
all the decoys based on the pair-wise structure similarity, and reports up to five models which
corresponds to the five largest structure clusters. The confidence of each model is quantitatively
measured by C-score that is calculated based on the significance of threading template alignments
and the convergence parameters of the structure assembly simulations.
C-score is typically in the range of [-5, 2], where a C-score of a higher value signifies a model with a higher 
confidence and vice-versa. TM-score and RMSD are estimated based on C-score and protein length following 
the correlation observed between these qualities. Since the top 5 models are ranked by the cluster size,
it is possible that the lower-rank models have a higher C-score in 
rare cases. Although the first model has a better
quality in most cases, it is also possible that the lower-rank models have a better quality than the
higher-rank models as seen in our benchmark tests. If the I-TASSER simulations converge, it
is possible to have less than 5 clusters generated; this is usually an indication that the
models have a good quality because of the converged simulations.)
<ul>
<li><a href=cscore.txt target="_blank">More about C-score</a>
<li><a href=lscore.html>Local structure accuracy profile of the top five models</a>
</ul>
</font>
<ul>   <div style="position:relative;left:1px;">
       <table  style="table-layout:fixed;width:1780px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
  <tr> <td colspan=5> <font size=2>(By right-click on the images, you can export image file or change the configurations, e.g. modifying the background color or stopping the spin of
your models)</font></td></tr>
       <tr>
       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model1.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin on;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin off", "spin off", "Spin On/Off", false)</script></td>
       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model2.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin off;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin off", "spin off", "Spin On/Off", false)</script></td>
       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model3.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin off;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin off", "spin off", "Spin On/Off", false)</script></td>
       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model4.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin off;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin off", "spin off", "Spin On/Off", false)</script></td>
       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model5.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin off;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin off", "spin off", "Spin On/Off", false)</script></td>
       </tr> <tr> <td  align =left>
<li><a href="model1.pdb" download>Download Model 1</a>

<li>C-score=-3.43 (<a href="cscore.txt" target="_blank" >Read more about C-score</a>) 

<li>Estimated TM-score = 0.34&plusmn;0.11

<li>Estimated RMSD = 13.9&plusmn;3.9&Aring;

<br><br>

</td>
 <td  align =left>
<li><a href="model2.pdb" download>Download Model 2</a>

<li>C-score = -2.84 <br><br>

</td>
 <td  align =left>
<li><a href="model3.pdb" download>Download Model 3</a>

<li>C-score = -3.72 <br><br>

</td>
 <td  align =left>
<li><a href="model4.pdb" download>Download Model 4</a>

<li>C-score = -4.56 <br><br>

</td>
 <td  align =left>
<li><a href="model5.pdb" download>Download Model 5</a>

<li>C-score = -4.79 <br><br>

</td>
       </tr>
       </table>
    </div>
    </div>
  </td></tr>
  </ul>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:670px;">
      <table>
         <tr><td style= "valign:middle;"><font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Proteins structurally close to the target in the PDB (as identified by <a href="http://zhanglab.ccmb.med.umich.edu/TM-align" target="_blank">TM-align</a>)</font></td></tr>
      </table>
   </div>
   <br/>
   
<font size=2>

(After the structure assembly simulation, I-TASSER uses the TM-align structural alignment program to match the first I-TASSER model to all structures in the PDB library. This section reports the top 10 proteins from the PDB that have the closest structural similarity, 

i.e. the highest <a href=http://zhanglab.ccmb.med.umich.edu/TM-score>TM-score</a>, to the predicted I-TASSER model. Due to the structural similarity, these proteins often have similar function to the target. However, users are encouraged to use the data in the next section 'Predicted function using COACH' to infer the function of the target protein, since COACH has been extensively trained to derive biological functions from multi-source of sequence and structure features which has on average a higher accuracy than the function annotations derived only from the global structure comparison.)</font> 

   <br/>

   <br/>

   <div>
   <div style="position:relative;left:30px;background:#F2F2F2;">
       <table  style="width:1200px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
           <tr>
           <td align ="center"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(400, "load model1_6w2vA.pdb; calculate structure rama; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;");</script><br/><SCRIPT>jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin on", "spin off", "Spin On/Off", false)</SCRIPT></td>
           <td valign="top"><b>Top 10 Identified stuctural analogs in PDB</b><br/><br/>
                 <table border="0" style="font-family:Arial;font-size:13px;">
                    <tr bgcolor="#DEDEDE"><th>Click<br>to view</th><th >Rank</th><th >PDB Hit</th><th >TM-score</th><th >RMSD<sup>a</sup></th><th >IDEN<sup>a</sup></th><th>Cov</th><th>Alignment</th></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_6w2vA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", true);</script></td><font face="Monospace">
                       <td valign="middle" align="center">1</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=6w2v" target="_blank">6w2vA</a></font></td><td align="center">0.836</td><td valign="middle" align="center"> 1.59</td><td valign="middle" align="center">0.111</td><td valign="middle">0.885</td><td align="center"><a href="model1_6w2vA.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_6w2wA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">2</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=6w2w" target="_blank">6w2wA</a></font></td><td align="center">0.730</td><td valign="middle" align="center"> 2.44</td><td valign="middle" align="center">0.090</td><td valign="middle">0.843</td><td align="center"><a href="model1_6w2wA.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_6w2rA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">3</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=6w2r" target="_blank">6w2rA</a></font></td><td align="center">0.675</td><td valign="middle" align="center"> 3.19</td><td valign="middle" align="center">0.102</td><td valign="middle">0.838</td><td align="center"><a href="model1_6w2rA.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_8flxA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">4</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=8flx" target="_blank">8flxA</a></font></td><td align="center">0.625</td><td valign="middle" align="center"> 4.06</td><td valign="middle" align="center">0.132</td><td valign="middle">0.851</td><td align="center"><a href="model1_8flxA.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_5cwiA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">5</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5cwi" target="_blank">5cwiA</a></font></td><td align="center">0.579</td><td valign="middle" align="center"> 3.59</td><td valign="middle" align="center">0.076</td><td valign="middle">0.719</td><td align="center"><a href="model1_5cwiA.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_1m56A.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">6</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1m56" target="_blank">1m56A</a></font></td><td align="center">0.547</td><td valign="middle" align="center"> 5.17</td><td valign="middle" align="center">0.047</td><td valign="middle">0.830</td><td align="center"><a href="model1_1m56A.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_7z10a.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">7</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=7z10" target="_blank">7z10a</a></font></td><td align="center">0.546</td><td valign="middle" align="center"> 5.14</td><td valign="middle" align="center">0.071</td><td valign="middle">0.830</td><td align="center"><a href="model1_7z10a.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_6z16A.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">8</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=6z16" target="_blank">6z16A</a></font></td><td align="center">0.546</td><td valign="middle" align="center"> 5.27</td><td valign="middle" align="center">0.086</td><td valign="middle">0.881</td><td align="center"><a href="model1_6z16A.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_3mk7A.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">9</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3mk7" target="_blank">3mk7A</a></font></td><td align="center">0.539</td><td valign="middle" align="center"> 5.20</td><td valign="middle" align="center">0.088</td><td valign="middle">0.843</td><td align="center"><a href="model1_3mk7A.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_8fbiA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">10</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=8fbi" target="_blank">8fbiA</a></font></td><td align="center">0.539</td><td valign="middle" align="center"> 3.86</td><td valign="middle" align="center">0.053</td><td valign="middle">0.702</td><td align="center"><a href="model1_8fbiA.pdb" download><font face="Arial">Download</font></a></td></tr>
               </table><br/>
               <table border="0" cellspacing="2" style="font-family:Arial;font-size:12px;background:#F2F2F2;">
                <tr><td valign="top">(a)</td><td align="justify">Query structure is shown in cartoon, while the structural analog is displayed using backbone trace.</td></tr>
                <tr><td valign="top">(b)</td><td align="justify">Ranking of proteins is based on TM-score of the structural alignment between the query structure and known structures in the PDB library.</td></tr>
                <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
                <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
                <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of the alignment by TM-align and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
               </table>
          </td></tr>
       </table>
    </div>
    </div>
  </td></tr>
  <tr><td>
  <br/><br/>
   <div style="background:#6599FF;width:450px;">
   <table><tr><td style= "valign:middle;">
      <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted function using <a href="http://zhanglab.ccmb.med.umich.edu/COFACTOR" target="_blank">COFACTOR</a> and <a href="http://zhanglab.ccmb.med.umich.edu/COACH" target="_blank">COACH</a></font>
   </td></tr></table>
   </div>
  <br/>
  
<font size=2>

(This section reports biological annotations of the target 
protein by COFACTOR and COACH based on the I-TASSER structure prediction. 
While COFACTOR deduces protein functions (ligand-binding sites, EC and GO)
using structure comparison and protein-protein networks,
COACH is a meta-server approach that combines multiple function
annotation results (on ligand-binding sites) from the COFACTOR, 
TM-SITE and S-SITE programs.)

</font>

<br/>

<br/>

  <tr><td>
   <div style="position:relative;left:30px;background:#FF9900; width:280px;">
      <table><tr><td  valign="middle;">
        <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Ligand binding sites</font>
      </td></tr></table>
   </div>
  <br/>
  <div style="position:relative;left:30px;background:#F2F2F2;width:100%">
       <table  style="width:1200px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
       <tr><td valign="top" align ="center"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(400, "load model1.pdb; calculate structure rama; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 178,179; wireframe 75; spacefill 100; color blue;select 178 and *.ca; label %m%R; color label magenta;select 179 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;     set spin x 10; set spin y 10; spin off;");</script><br/><SCRIPT> jmolButton("Reset","Reset to initial orientation");  jmolCheckbox("set spin x 10; set spin y 10; spin on", "spin off", "Spin On/Off", false)</SCRIPT>
</td>
       <td valign="top">       <table border="0"  style="font-family:Arial;font-size:13px;"><tr bgcolor="#DEDEDE"><th>Click<br>to view</th><th>Rank</th><th>C-score</th><th>Cluster<br>size</th> <th>PDB<br>Hit</th> <th>Lig<br>Name</th> <th>Download<br>Complex</th> <th align=left>Ligand Binding Site Residues</th> </tr>
       <tr >
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 178,179; color blue; wireframe 75; spacefill 100;select 178 and *.ca; label %m%R; color label magenta;select 179 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", true);</script></td>
    <font face="Monospace"><td valign="middle" align="center">1</td><td valign="middle" align="center">0.07</td> <td valign="middle" align="center">5</td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch_pdb.cgi?pdbid=2pk0 target="_blank">2pk0A</a></td>  <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=MG target="_blank">MG</a></td>      <td valign="middle" align=center>N/A</td>     <td valign="middle" align="left">178,179</td>    </tr>
       <tr bgcolor="#DEDEDE">
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_TM_4h0lA_BS06_CLA.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 188,191,192,195,218,222; color blue; wireframe 75; spacefill 100;select 188 and *.ca; label %m%R; color label magenta;select 191 and *.ca; label %m%R; color label magenta;select 192 and *.ca; label %m%R; color label magenta;select 195 and *.ca; label %m%R; color label magenta;select 218 and *.ca; label %m%R; color label magenta;select 222 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>
    <font face="Monospace"><td valign="middle" align="center">2</td><td valign="middle" align="center">0.07</td> <td valign="middle" align="center">4</td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch_pdb.cgi?pdbid=4h0l target="_blank">4h0lA</a></td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=CLA target="_blank">CLA</a> </td>  <td valign="middle" align=center><a href="CH_TM_4h0lA_BS06_CLA.pdb" download>Rep</a>, <a href="CH_complex2.pdb" download>Mult</a></td>  <td valign="middle" align="left">188,191,192,195,218,222</td>    </tr>
       <tr >
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_CF_3fyeA_BS06_DMU.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 154,156,182,183,186,187,190; color blue; wireframe 75; spacefill 100;select 154 and *.ca; label %m%R; color label magenta;select 156 and *.ca; label %m%R; color label magenta;select 182 and *.ca; label %m%R; color label magenta;select 183 and *.ca; label %m%R; color label magenta;select 186 and *.ca; label %m%R; color label magenta;select 187 and *.ca; label %m%R; color label magenta;select 190 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>
    <font face="Monospace"><td valign="middle" align="center">3</td><td valign="middle" align="center">0.03</td> <td valign="middle" align="center">2</td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch_pdb.cgi?pdbid=3fye target="_blank">3fyeA</a></td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=DMU target="_blank">DMU</a> </td>  <td valign="middle" align=center><a href="CH_CF_3fyeA_BS06_DMU.pdb" download>Rep</a>, <a href="CH_complex3.pdb" download>Mult</a></td>  <td valign="middle" align="left">154,156,182,183,186,187,190</td>    </tr>
       <tr bgcolor="#DEDEDE">
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 171,177; color blue; wireframe 75; spacefill 100;select 171 and *.ca; label %m%R; color label magenta;select 177 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>
    <font face="Monospace"><td valign="middle" align="center">4</td><td valign="middle" align="center">0.03</td> <td valign="middle" align="center">2</td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch_pdb.cgi?pdbid=3guj target="_blank">3gujA</a></td>  <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=CO3 target="_blank">CO3</a></td>      <td valign="middle" align=center>N/A</td>     <td valign="middle" align="left">171,177</td>    </tr>
       <tr >
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_TM_3b71A_BS01_III.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 78,80,156,159,163; color blue; wireframe 75; spacefill 100;select 78 and *.ca; label %m%R; color label magenta;select 80 and *.ca; label %m%R; color label magenta;select 156 and *.ca; label %m%R; color label magenta;select 159 and *.ca; label %m%R; color label magenta;select 163 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>
    <font face="Monospace"><td valign="middle" align="center">5</td><td valign="middle" align="center">0.03</td> <td valign="middle" align="center">2</td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch_pdb.cgi?pdbid=3b71 target="_blank">3b71A</a></td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=III target="_blank">PEPTIDE</a> </td>  <td valign="middle" align=center><a href="CH_TM_3b71A_BS01_III.pdb" download>Rep</a>, <a href="CH_complex5.pdb" download>Mult</a></td>  <td valign="middle" align="left">78,80,156,159,163</td>    </tr>
       </table>
       <br/>
       <table  cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
       <tr><td valign="top"></td><td align="justify"><a href=prob.txt target="_blank">Download</a> the residue-specific ligand binding probability, which is estimated by SVM. </td></tr> 
       <tr><td valign="top"></td><td align="justify"><a href=Bsites.inf target="_blank">Download</a> the all possible binding ligands and detailed prediction summary. </td></tr> 
       <tr><td valign="top"></td><td align="justify"><a href=Bsites.clr target="_blank">Download</a> the templates clustering results. </td></tr> 
        <tr><td valign="top">(a)</td><td align="justify"><b>C-score</b> is the confidence score of the prediction. C-score ranges [0-1], where a higher score indicates a more reliable prediction. </td></tr>
        <tr><td valign="top">(b)</td><td align="justify"><b>Cluster size</b> is the total number of templates in a cluster. </td></tr>
       <tr><td valign="top">(c)</td><td align="justify"><b>Lig Name</b> is name of possible binding ligand. Click the name to view its information in <a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/>the BioLiP database</a>.</td></tr>
       <tr><td valign="top">(d)</td><td align="justify"><b>Rep</b> is a single complex structure with the most representative ligand in the cluster, i.e., the one listed in the <b>Lig Name</b> column. <br>
 <b>Mult</b> is the complex structures with all potential binding ligands in the cluster.</td></tr>
<br>       </table></td></tr>
     </table>
  </div>
</td></tr>
  <tr><td>
  <br/>
   <div style="position:relative;left:30px;background:#FF9900; width:430px;">
   <table><tr><td style= "valign:middle;">
        <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Enzyme Commission (EC) numbers and active sites</font>
   </td></tr></table>
   </div>
   <br/>
  <div style="position:relative;left:30px;background:#F2F2F2;width:100%">
       <table  style="width:1200px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
       <tr><td align ="center"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(400, "load model1.pdb; calculate structure rama; set frank off; select all; spacefill off; wireframe off; cartoons; select 97; wireframe 70; spacefill 100; color cpk;  select 97 and *.ca; label %m%R; color label magenta; select 131; wireframe 70; spacefill 100; color cpk;  select 131 and *.ca; label %m%R; color label magenta; select 169; wireframe 70; spacefill 100; color cpk;  select 169 and *.ca; label %m%R; color label magenta;      set spin x 10; set spin y 10; spin off;");</script><br/><SCRIPT>jmolButton("Reset","Reset to initial orientation");  jmolCheckbox("spin on", "spin off", "Spin On/Off", false)</SCRIPT></td>
           <td valign="top">
             <table  border="0" style="font-family:Arial;font-size:13px;"><tr bgcolor="#DEDEDE"> <th>Click<br>to view</th><th >Rank</th><th >Cscore<sup>EC</sup></th><th>PDB<br/>Hit</th><th >TM-score</th><th >RMSD<sup>a</sup></th><th >IDEN<sup>a</sup></th><th >Cov</th><th >EC Number</th><th align=left>Active Site Residues</th></tr>
         <tr >
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off; select 97; wireframe 70; spacefill 100; color cpk;  select 97 and *.ca; label %m%R; color label magenta; select 131; wireframe 70; spacefill 100; color cpk;  select 131 and *.ca; label %m%R; color label magenta; select 169; wireframe 70; spacefill 100; color cpk;  select 169 and *.ca; label %m%R; color label magenta;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">1</font></td><td align="center"><font face="Arial">0.087</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1i5n" target="_blank">1i5nD</a></td><td align="center"><font face="Arial">0.426</font></td><td align="center"><font face="Arial"> 2.76</font></td><td align="center"><font face="Arial">0.074</font></td><td align="right"><font face="Arial">0.506</font></td>
         <td align="left"><span title="Histidine kinase."><a href="http://ca.expasy.org/enzyme/2.7.13.3" target="_blank"><font face="Arial">2.7.13.3</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">97,131,169</font></td>         </tr>
         <tr bgcolor="#DEDEDE">
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">2</font></td><td align="center"><font face="Arial">0.086</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1m56" target="_blank">1m56A</a></td><td align="center"><font face="Arial">0.547</font></td><td align="center"><font face="Arial"> 5.17</font></td><td align="center"><font face="Arial">0.047</font></td><td align="right"><font face="Arial">0.830</font></td>
         <td align="left"><span title="Cytochrome-c oxidase."><a href="http://ca.expasy.org/enzyme/1.9.3.1" target="_blank"><font face="Arial">1.9.3.1</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">NA</font></td>         </tr>
         <tr >
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">3</font></td><td align="center"><font face="Arial">0.080</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1xme" target="_blank">1xmeA</a></td><td align="center"><font face="Arial">0.491</font></td><td align="center"><font face="Arial"> 5.71</font></td><td align="center"><font face="Arial">0.047</font></td><td align="right"><font face="Arial">0.821</font></td>
         <td align="left"><span title="Cytochrome-c oxidase."><a href="http://ca.expasy.org/enzyme/1.9.3.1" target="_blank"><font face="Arial">1.9.3.1</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">NA</font></td>         </tr>
         <tr bgcolor="#DEDEDE">
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off; select 80; wireframe 70; spacefill 100; color cpk;  select 80 and *.ca; label %m%R; color label magenta; select 97; wireframe 70; spacefill 100; color cpk;  select 97 and *.ca; label %m%R; color label magenta; select 161; wireframe 70; spacefill 100; color cpk;  select 161 and *.ca; label %m%R; color label magenta;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">4</font></td><td align="center"><font face="Arial">0.079</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1tqg" target="_blank">1tqgA</a></td><td align="center"><font face="Arial">0.362</font></td><td align="center"><font face="Arial"> 2.88</font></td><td align="center"><font face="Arial">0.069</font></td><td align="right"><font face="Arial">0.417</font></td>
         <td align="left"><span title="Histidine kinase."><a href="http://ca.expasy.org/enzyme/2.7.13.3" target="_blank"><font face="Arial">2.7.13.3</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">80,97,161</font></td>         </tr>
         <tr >
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off; select 146; wireframe 70; spacefill 100; color cpk;  select 146 and *.ca; label %m%R; color label magenta;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">5</font></td><td align="center"><font face="Arial">0.078</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3i4l" target="_blank">3i4lA</a></td><td align="center"><font face="Arial">0.487</font></td><td align="center"><font face="Arial"> 5.61</font></td><td align="center"><font face="Arial">0.080</font></td><td align="right"><font face="Arial">0.791</font></td>
         <td align="left"><span title="H(+)-transporting two-sector ATPase."><a href="http://ca.expasy.org/enzyme/3.6.3.14" target="_blank"><font face="Arial">3.6.3.14</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">146</font></td>         </tr>
         </table><br/>
         <table border="0"  cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
          <tr><td valign="top">&nbsp;</td><td align="justify"><b>Click on the radio buttons to visualize predicted active site residues.</b></td></tr>
           <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>EC</sup> is the confidence score for the EC number prediction. Cscore<sup>EC</sup> values range in between [0-1]; <br>where a higher score indicates a more reliable EC number prediction.</td></tr>
           <tr><td valign="top">(b)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>
           <tr><td valign="bottom">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
           <tr><td valign="bottom">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
           <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of global structural  alignment and is equal to the number of structurally aligned residues divided <br>by length of the query protein.</td></tr>
         </table>
        </td></tr></table>
  </div>
</td></tr>
  <tr><td>
  <br/>
   <div style="position:relative;left:30px;background:#FF9900; width:280px;">
      <table><tr><td  valign="middle;">
        <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Gene Ontology (GO) terms</font>
      </td></tr></table>
   </div>
  <div style="position:relative;left:30px;background:#F2F2F2;width:100%">
       <table  style="width:1200px;font-size:13px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
      <tr background:#FF9900;><th colspan=8 align="left"><font face="Arial" size="3">Top 10 homologous GO templates in PDB</font><th><td>&nbsp;</td></tr>
  <tr bgcolor="#DEDEDE"><th ><font face="Arial">Rank</font></th><th><font face="Arial">Cscore<sup>GO</sup></font></th><th><font face="Arial">TM-score</font></th><th><font face="Arial">RMSD<sup>a</sup></font></th><th><font face="Arial">IDEN<sup>a</sup></font></th><th><font face="Arial">Cov</font></th><th align="left"><font face="Arial">PDB Hit</font></th><th align="left"><font face="Arial">Associated GO Terms</font></th></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">1</font></td><td align="center"  valign="middle"><font face="Arial"> 0.15</font></td><td  valign="middle" align="center"><font face="Arial">0.5004</font></td><td align="center"  valign="middle"><font face="Arial"> 5.49</font></td><td align="center"  valign="middle"><font face="Arial"> 0.04</font></td><td  valign="middle" align="center"><font face="Arial"> 0.81</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3o0r" target="_blank">3o0rB</a></td><td valign="middle"><font face="Arial">
<span title="membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016020 target="_blank">GO:0016020</a></span>
<span title="plasma membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005886 target="_blank">GO:0005886</a></span>
<span title="integral to membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016021 target="_blank">GO:0016021</a></span>
<span title="oxidation-reduction process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0055114 target="_blank">GO:0055114</a></span>
<span title="respiratory chain"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0070469 target="_blank">GO:0070469</a></span>
<span title="oxidoreductase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016491 target="_blank">GO:0016491</a></span>
<span title="transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006810 target="_blank">GO:0006810</a></span>
<span title="nitric oxide reductase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016966 target="_blank">GO:0016966</a></span>
<span title="electron transport chain"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0022900 target="_blank">GO:0022900</a></span>
<span title="metal ion binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046872 target="_blank">GO:0046872</a></span>
<span title="iron ion binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005506 target="_blank">GO:0005506</a></span>
<span title="electron carrier activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009055 target="_blank">GO:0009055</a></span>
<span title="aerobic respiration"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009060 target="_blank">GO:0009060</a></span>
<span title="heme binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0020037 target="_blank">GO:0020037</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">2</font></td><td align="center"  valign="middle"><font face="Arial"> 0.13</font></td><td  valign="middle" align="center"><font face="Arial">0.5474</font></td><td align="center"  valign="middle"><font face="Arial"> 5.17</font></td><td align="center"  valign="middle"><font face="Arial"> 0.05</font></td><td  valign="middle" align="center"><font face="Arial"> 0.83</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1m56" target="_blank">1m56A</a></td><td valign="middle"><font face="Arial">
<span title="respiratory chain"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0070469 target="_blank">GO:0070469</a></span>
<span title="metal ion binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046872 target="_blank">GO:0046872</a></span>
<span title="ion transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006811 target="_blank">GO:0006811</a></span>
<span title="heme binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0020037 target="_blank">GO:0020037</a></span>
<span title="oxidation-reduction process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0055114 target="_blank">GO:0055114</a></span>
<span title="proton transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015992 target="_blank">GO:0015992</a></span>
<span title="electron carrier activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009055 target="_blank">GO:0009055</a></span>
<span title="copper ion binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005507 target="_blank">GO:0005507</a></span>
<span title="oxidoreductase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016491 target="_blank">GO:0016491</a></span>
<span title="electron transport chain"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0022900 target="_blank">GO:0022900</a></span>
<span title="plasma membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005886 target="_blank">GO:0005886</a></span>
<span title="integral to membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016021 target="_blank">GO:0016021</a></span>
<span title="transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006810 target="_blank">GO:0006810</a></span>
<span title="iron ion binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005506 target="_blank">GO:0005506</a></span>
<span title="membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016020 target="_blank">GO:0016020</a></span>
<span title="aerobic respiration"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009060 target="_blank">GO:0009060</a></span>
<span title="cytochrome-c oxidase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0004129 target="_blank">GO:0004129</a></span>
</font></td></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">3</font></td><td align="center"  valign="middle"><font face="Arial"> 0.12</font></td><td  valign="middle" align="center"><font face="Arial">0.5051</font></td><td align="center"  valign="middle"><font face="Arial"> 6.09</font></td><td align="center"  valign="middle"><font face="Arial"> 0.05</font></td><td  valign="middle" align="center"><font face="Arial"> 0.88</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1occ" target="_blank">1occA</a></td><td valign="middle"><font face="Arial">
<span title="mitochondrial inner membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005743 target="_blank">GO:0005743</a></span>
<span title="heme binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0020037 target="_blank">GO:0020037</a></span>
<span title="electron carrier activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009055 target="_blank">GO:0009055</a></span>
<span title="oxidation-reduction process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0055114 target="_blank">GO:0055114</a></span>
<span title="electron transport chain"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0022900 target="_blank">GO:0022900</a></span>
<span title="oxidoreductase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016491 target="_blank">GO:0016491</a></span>
<span title="metal ion binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046872 target="_blank">GO:0046872</a></span>
<span title="respiratory chain"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0070469 target="_blank">GO:0070469</a></span>
<span title="mitochondrion"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005739 target="_blank">GO:0005739</a></span>
<span title="aerobic respiration"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009060 target="_blank">GO:0009060</a></span>
<span title="transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006810 target="_blank">GO:0006810</a></span>
<span title="iron ion binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005506 target="_blank">GO:0005506</a></span>
<span title="cytochrome-c oxidase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0004129 target="_blank">GO:0004129</a></span>
<span title="integral to membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016021 target="_blank">GO:0016021</a></span>
<span title="membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016020 target="_blank">GO:0016020</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">4</font></td><td align="center"  valign="middle"><font face="Arial"> 0.12</font></td><td  valign="middle" align="center"><font face="Arial">0.5393</font></td><td align="center"  valign="middle"><font face="Arial"> 5.20</font></td><td align="center"  valign="middle"><font face="Arial"> 0.09</font></td><td  valign="middle" align="center"><font face="Arial"> 0.84</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3mk7" target="_blank">3mk7A</a></td><td valign="middle"><font face="Arial">
<span title="cytochrome-c oxidase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0004129 target="_blank">GO:0004129</a></span>
<span title="iron ion binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005506 target="_blank">GO:0005506</a></span>
<span title="electron carrier activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009055 target="_blank">GO:0009055</a></span>
<span title="aerobic respiration"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009060 target="_blank">GO:0009060</a></span>
<span title="integral to membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016021 target="_blank">GO:0016021</a></span>
<span title="heme binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0020037 target="_blank">GO:0020037</a></span>
<span title="plasma membrane respiratory chain complex IV"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0045278 target="_blank">GO:0045278</a></span>
<span title="oxidation-reduction process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0055114 target="_blank">GO:0055114</a></span>
</font></td></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">5</font></td><td align="center"  valign="middle"><font face="Arial"> 0.08</font></td><td  valign="middle" align="center"><font face="Arial">0.5045</font></td><td align="center"  valign="middle"><font face="Arial"> 4.70</font></td><td align="center"  valign="middle"><font face="Arial"> 0.05</font></td><td  valign="middle" align="center"><font face="Arial"> 0.74</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3d19" target="_blank">3d19A</a></td><td valign="middle"><font face="Arial">
<span title="metal ion binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046872 target="_blank">GO:0046872</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">6</font></td><td align="center"  valign="middle"><font face="Arial"> 0.08</font></td><td  valign="middle" align="center"><font face="Arial">0.4914</font></td><td align="center"  valign="middle"><font face="Arial"> 5.71</font></td><td align="center"  valign="middle"><font face="Arial"> 0.05</font></td><td  valign="middle" align="center"><font face="Arial"> 0.82</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1xme" target="_blank">1xmeA</a></td><td valign="middle"><font face="Arial">
<span title="oxidation-reduction process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0055114 target="_blank">GO:0055114</a></span>
<span title="electron carrier activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009055 target="_blank">GO:0009055</a></span>
<span title="transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006810 target="_blank">GO:0006810</a></span>
<span title="integral to membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016021 target="_blank">GO:0016021</a></span>
<span title="proton transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015992 target="_blank">GO:0015992</a></span>
<span title="heme binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0020037 target="_blank">GO:0020037</a></span>
<span title="aerobic respiration"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009060 target="_blank">GO:0009060</a></span>
<span title="oxidoreductase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016491 target="_blank">GO:0016491</a></span>
<span title="electron transport chain"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0022900 target="_blank">GO:0022900</a></span>
<span title="ion transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006811 target="_blank">GO:0006811</a></span>
<span title="iron ion binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005506 target="_blank">GO:0005506</a></span>
<span title="metal ion binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046872 target="_blank">GO:0046872</a></span>
<span title="plasma membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005886 target="_blank">GO:0005886</a></span>
<span title="cytochrome-c oxidase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0004129 target="_blank">GO:0004129</a></span>
<span title="respiratory chain"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0070469 target="_blank">GO:0070469</a></span>
<span title="membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016020 target="_blank">GO:0016020</a></span>
</font></td></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">7</font></td><td align="center"  valign="middle"><font face="Arial"> 0.08</font></td><td  valign="middle" align="center"><font face="Arial">0.4971</font></td><td align="center"  valign="middle"><font face="Arial"> 4.37</font></td><td align="center"  valign="middle"><font face="Arial"> 0.07</font></td><td  valign="middle" align="center"><font face="Arial"> 0.67</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1st6" target="_blank">1st6A</a></td><td valign="middle"><font face="Arial">
<span title="epithelial cell-cell adhesion"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0090136 target="_blank">GO:0090136</a></span>
<span title="cell-cell junction"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005911 target="_blank">GO:0005911</a></span>
<span title="protein localization at cell surface"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0034394 target="_blank">GO:0034394</a></span>
<span title="lamellipodium assembly"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0030032 target="_blank">GO:0030032</a></span>
<span title="cytoskeleton"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005856 target="_blank">GO:0005856</a></span>
<span title="cell adhesion"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0007155 target="_blank">GO:0007155</a></span>
<span title="apical junction assembly"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0043297 target="_blank">GO:0043297</a></span>
<span title="cell-cell adherens junction"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005913 target="_blank">GO:0005913</a></span>
<span title="morphogenesis of an epithelium"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0002009 target="_blank">GO:0002009</a></span>
<span title="cadherin binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0045296 target="_blank">GO:0045296</a></span>
<span title="costamere"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0043034 target="_blank">GO:0043034</a></span>
<span title="cytoplasm"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005737 target="_blank">GO:0005737</a></span>
<span title="beta-dystroglycan binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0002166 target="_blank">GO:0002166</a></span>
<span title="cell junction"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0030054 target="_blank">GO:0030054</a></span>
<span title="adherens junction assembly"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0034333 target="_blank">GO:0034333</a></span>
<span title="actin binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003779 target="_blank">GO:0003779</a></span>
<span title="mitochondrial inner membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005743 target="_blank">GO:0005743</a></span>
<span title="plasma membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005886 target="_blank">GO:0005886</a></span>
<span title="fascia adherens"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005916 target="_blank">GO:0005916</a></span>
<span title="protein complex"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0043234 target="_blank">GO:0043234</a></span>
<span title="beta-catenin binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0008013 target="_blank">GO:0008013</a></span>
<span title="protein binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005515 target="_blank">GO:0005515</a></span>
<span title="alpha-catenin binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0045294 target="_blank">GO:0045294</a></span>
<span title="membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016020 target="_blank">GO:0016020</a></span>
<span title="adherens junction"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005912 target="_blank">GO:0005912</a></span>
<span title="regulation of cell migration"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0030334 target="_blank">GO:0030334</a></span>
<span title="focal adhesion"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005925 target="_blank">GO:0005925</a></span>
<span title="structural molecule activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005198 target="_blank">GO:0005198</a></span>
<span title="actin cytoskeleton"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015629 target="_blank">GO:0015629</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">8</font></td><td align="center"  valign="middle"><font face="Arial"> 0.08</font></td><td  valign="middle" align="center"><font face="Arial">0.4846</font></td><td align="center"  valign="middle"><font face="Arial"> 5.51</font></td><td align="center"  valign="middle"><font face="Arial"> 0.12</font></td><td  valign="middle" align="center"><font face="Arial"> 0.79</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3gqb" target="_blank">3gqbA</a></td><td valign="middle"><font face="Arial">
<span title="proton-transporting two-sector ATPase complex, catalytic domain"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0033178 target="_blank">GO:0033178</a></span>
<span title="hydrolase activity, acting on acid anhydrides, catalyzing transmembrane movement of substances"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016820 target="_blank">GO:0016820</a></span>
<span title="proton-transporting ATPase activity, rotational mechanism"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046961 target="_blank">GO:0046961</a></span>
<span title="hydrogen ion transporting ATP synthase activity, rotational mechanism"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046933 target="_blank">GO:0046933</a></span>
<span title="ATP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005524 target="_blank">GO:0005524</a></span>
<span title="proton-transporting two-sector ATPase complex"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016469 target="_blank">GO:0016469</a></span>
<span title="transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006810 target="_blank">GO:0006810</a></span>
<span title="proton transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015992 target="_blank">GO:0015992</a></span>
<span title="ATP metabolic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046034 target="_blank">GO:0046034</a></span>
<span title="ATP biosynthetic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006754 target="_blank">GO:0006754</a></span>
<span title="ion transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006811 target="_blank">GO:0006811</a></span>
<span title="nucleotide binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000166 target="_blank">GO:0000166</a></span>
<span title="hydrolase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016787 target="_blank">GO:0016787</a></span>
<span title="ATP hydrolysis coupled proton transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015991 target="_blank">GO:0015991</a></span>
</font></td></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">9</font></td><td align="center"  valign="middle"><font face="Arial"> 0.08</font></td><td  valign="middle" align="center"><font face="Arial">0.4865</font></td><td align="center"  valign="middle"><font face="Arial"> 5.25</font></td><td align="center"  valign="middle"><font face="Arial"> 0.05</font></td><td  valign="middle" align="center"><font face="Arial"> 0.77</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1hf8" target="_blank">1hf8A</a></td><td valign="middle"><font face="Arial">
<span title="phospholipid binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005543 target="_blank">GO:0005543</a></span>
<span title="1-phosphatidylinositol binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005545 target="_blank">GO:0005545</a></span>
<span title="clathrin coat"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0030118 target="_blank">GO:0030118</a></span>
<span title="clathrin binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0030276 target="_blank">GO:0030276</a></span>
<span title="clathrin coat assembly"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0048268 target="_blank">GO:0048268</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">10</font></td><td align="center"  valign="middle"><font face="Arial"> 0.08</font></td><td  valign="middle" align="center"><font face="Arial">0.4847</font></td><td align="center"  valign="middle"><font face="Arial"> 5.30</font></td><td align="center"  valign="middle"><font face="Arial"> 0.08</font></td><td  valign="middle" align="center"><font face="Arial"> 0.76</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3m4y" target="_blank">3m4yA</a></td><td valign="middle"><font face="Arial">
<span title="nucleotide binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000166 target="_blank">GO:0000166</a></span>
<span title="ATP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005524 target="_blank">GO:0005524</a></span>
<span title="ATP synthesis coupled proton transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015986 target="_blank">GO:0015986</a></span>
<span title="ATP hydrolysis coupled proton transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015991 target="_blank">GO:0015991</a></span>
<span title="proton transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015992 target="_blank">GO:0015992</a></span>
<span title="proton-transporting two-sector ATPase complex"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016469 target="_blank">GO:0016469</a></span>
<span title="hydrolase activity, acting on acid anhydrides, catalyzing transmembrane movement of substances"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016820 target="_blank">GO:0016820</a></span>
<span title="nucleoside-triphosphatase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0017111 target="_blank">GO:0017111</a></span>
<span title="proton-transporting two-sector ATPase complex, catalytic domain"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0033178 target="_blank">GO:0033178</a></span>
<span title="ATP metabolic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046034 target="_blank">GO:0046034</a></span>
<span title="hydrogen ion transporting ATP synthase activity, rotational mechanism"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046933 target="_blank">GO:0046933</a></span>
<span title="proton-transporting ATPase activity, rotational mechanism"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046961 target="_blank">GO:0046961</a></span>
</font></td></tr>
   </table>
<p>
       <table  style="width:1200px;font-size:13px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
     <tr ><th colspan=6 align="left"><font face="Arial" size="3"><br>Consensus prediction of GO terms</font><th><td>&nbsp;</td></tr>
      <tr><td>
      <table cellpadding="2" cellspacing="3" style="font-family:Monospace;font-size:13px;background:#F2F2F2">
       <tr bgcolor="#DEDEDE"><td><font face="Arial"><b>Molecular Function</b></font></td>
      <td align="center"><font face="Arial"><span title="heme binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0020037 target="_blank">GO:0020037</a></span></font></td>
      <td align="center"><font face="Arial"><span title="electron carrier activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009055 target="_blank">GO:0009055</a></span></font></td>
      <td align="center"><font face="Arial"><span title="cytochrome-c oxidase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0004129 target="_blank">GO:0004129</a></span></font></td>
      <td align="center"><font face="Arial"><span title="oxidoreductase activity, acting on other nitrogenous compounds as donors"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016661 target="_blank">GO:0016661</a></span></font></td>
</tr>
<tr><td ><font face="Arial"><b>GO-Score</b></font></td>      <td align="center"><font face="Arial">0.42</font></td>
      <td align="center"><font face="Arial">0.42</font></td>
      <td align="center"><font face="Arial">0.32</font></td>
      <td align="center"><font face="Arial">0.31</font></td>
  </tr>
       <tr bgcolor="#DEDEDE"><td><font face="Arial"><b>Biological Process</b></font></td>
      <td align="center"><font face="Arial"><span title="aerobic respiration"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009060 target="_blank">GO:0009060</a></span></font></td>
      <td align="center"><font face="Arial"><span title="transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006810 target="_blank">GO:0006810</a></span></font></td>
      <td align="center"><font face="Arial"><span title="electron transport chain"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0022900 target="_blank">GO:0022900</a></span></font></td>
</tr>
<tr><td ><font face="Arial"><b>GO-Score</b></font></td>      <td align="center"><font face="Arial">0.42</font></td>
      <td align="center"><font face="Arial">0.35</font></td>
      <td align="center"><font face="Arial">0.35</font></td>
  </tr>
       <tr bgcolor="#DEDEDE"><td><font face="Arial"><b>Cellular Component</b></font></td>
      <td align="center"><font face="Arial"><span title="cell periphery"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0071944 target="_blank">GO:0071944</a></span></font></td>
      <td align="center"><font face="Arial"><span title="integral to membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016021 target="_blank">GO:0016021</a></span></font></td>
      <td align="center"><font face="Arial"><span title="respiratory chain"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0070469 target="_blank">GO:0070469</a></span></font></td>
</tr>
<tr><td ><font face="Arial"><b>GO-Score</b></font></td>      <td align="center"><font face="Arial">0.52</font></td>
      <td align="center"><font face="Arial">0.42</font></td>
      <td align="center"><font face="Arial">0.35</font></td>
  </tr>
   </table>
  </td></tr></table>
<p>
       <table  style="width:1200px;font-size:12px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
      <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is a combined measure for evaluating global and local similarity between query and template protein. It's range is [0-1] and higher values indicate more confident predictions.</td></tr>
      <tr><td valign="top">(b)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>
      <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
      <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
      <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of global structural alignment and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
      <tr><td valign="top">(f)</td><td align="justify">The second table shows a consensus GO terms amongst the top scoring templates. The GO-Score associated with each prediction is defined as the average weight of the GO term, where the weights are assigned based on Cscore<sup>GO</sup> of the template.</td></tr>
     </table>
  </div>
</td></tr>
<tr><td><br/></td></tr>
<tr><td>
<h3 align="center">[Click on <a href="S760663_results.tar.bz2" download>S760663_results.tar.bz2</a> to download the tarball file including all modeling results listed on this page] </h3>
<hr><br/>
<div style="position:relative;left:5px;">

<!-- please change citations at
  /nfs/amino-home/zhanglabs/I-TASSER/bin/OUTPUTmod_ResQ
  /expanse/community/itasser/I-TASSER/bin/OUTPUTmod_ResQ
  -->
  
<font size=3>
Please cite the following articles when you use the I-TASSER server:
<ul>
<li>
  Wei Zheng, Chengxin Zhang, Yang Li, Robin Pearce, Eric W. Bell, Yang Zhang.
  Folding non-homology proteins by coupling deep-learning contact maps with I-TASSER assembly simulations.
  Cell Reports Methods, 1: 100014 (2021).
<li>
  Chengxin Zhang, Peter L. Freddolino, and Yang Zhang.
  COFACTOR: improved protein function prediction by combining structure,
  sequence and protein-protein interaction information.
  Nucleic Acids Research, 45: W291-299 (2017).
<li>
  Jianyi Yang, Yang Zhang. 
  I-TASSER server: new development for protein structure and function predictions,
  Nucleic Acids Research, 43: W174-W181, 2015.
</ul>
</font>

</div><br></td></tr>
</table></body>
</html>
